test/e2e/proxy_test.go

```diff
@@ -217,7 +223,7 @@ func TestProxyFlow(t *testing.T) {
 				w := newWsWatcher(t, *user, user.compliantUsername, proxyWorkspaceURL)
 				closeConnection := w.Start()
 				defer closeConnection()
-				proxyCl, err := hostAwait.CreateAPIProxyClient(t, user.token, proxyWorkspaceURL) // proxy client with workspace context
+				workspaceCl, err := hostAwait.CreateAPIProxyClient(t, user.token, proxyWorkspaceURL) // proxy client with workspace context
```

Comment:
minor detail - I see in the other test cases this object is named `proxyCL`, probably a better name given the function call `CreateAPIProxyClient`

Reply:
It's intentional because it's specifically using the workspace context `proxyWorkspaceURL` and not the base proxy URL

Reply:
Ok I see it adds the `workspaces` string in the path of the URL.

Sorry I was just trying to understand the difference compared to those:
https://github.com/rajivnathan/toolchain-e2e/blob/proxyrequestcheck/test/e2e/proxy_test.go#L209
https://github.com/rajivnathan/toolchain-e2e/blob/0232d856fc6239ad7fc0ebd6749d091bdaae0113/test/e2e/proxy_test.go#L280